app/controllers/Application.java

```diff
@@ -144,6 +144,22 @@ public static Result dashboard() {
   }
 
   /**
+   * Returns most recent flowExecId that is like value which can use % and _ SQL wild cards.
+   */
+  private static String bestMatchForFlowExecIdLikeValue(String value) {
+    List<AppResult> matches = AppResult.find
+            .select(AppResult.TABLE.FLOW_EXEC_ID)
+            .where().like(AppResult.TABLE.FLOW_DEF_ID, value)
```

Comment:
What if it matches with  multiple flow def id ? You might want to do a distinct on flow_def_id first to make sure there is only one flow def with that name. Then get the exec id with the most recent finish time.